### PR TITLE
Indx page

### PR DESCRIPTION
**Pull request by rgrupesh**
How It Works image added

Related #31

### CODE REVIEW

**Comment by rgrupesh** [comment]
https://deploy-preview-38--karmada.netlify.app/

**Comment by RainbowMango** [comment]
Looks pretty good.

Can you add two more commits on it to deal with `Why Karmada` and `Supporters`?

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by RainbowMango** [comment]
Great! @rgrupesh . We are more close now.

File: src/pages/index.js [hunk 1]
```diff
@@ -87,6 +87,12 @@ export default function Home() {
           </h3>
         <div className={styles.heroHowItWorks}>
         <img className={styles.heroLogo} src={useBaseUrl('img/how-it-works.png')} alt="Karmada Control Plane" />
+        </div>
+        <h3 className="hero__subtitle">
+            <Translate>Supporters</Translate>
+          </h3>
+        <div className={styles.heroSupporter}>
+        <img className={styles.heroLogo} src={useBaseUrl('img/supporters.png')} alt="Karmada Supporters" />
```
Comment (RainbowMango) [inline src/pages/index.js:95]: Are you putting all the supporters into one image? I don't think we should. I don't want to edit the image every time a new supporter joined.

That explains why the picture looks indistinct.

Comment (rgrupesh) [reply src/pages/index.js:95]: In my opinion, the overall flow of the website looks good when we put all the supporters into one image. We could change that. Personally, I prefer combination into one image. Please, let me know what you think.

**Comment by Arhell** [comment]
@RainbowMango not critical, I'll fix it anyway when everything is in the main branch or in another

File: src/pages/index.js (hunk 1 above)
Comment (rgrupesh) [reply src/pages/index.js:100]: Anything else that you would like to see on index page through this PR? @RainbowMango

Comment (RainbowMango) [reply src/pages/index.js:100]: `why Karmada` section looks unordered?

Can we organize the content by the table?

Comment (rgrupesh) [reply src/pages/index.js:100]: `Why Karmada` section is defined that way.
Screenshot from 2022-04-18 17-57-01

Comment (RainbowMango) [reply src/pages/index.js:100]: Can we add some dummy images provisionally? for good-looking.

Comment (rgrupesh) [reply src/pages/index.js:100]: I tried doing that. It looks something like this:

Screenshot from 2022-04-19 15-01-18

 We could use different image for each of them.

Comment (RainbowMango) [reply src/pages/index.js:100]: That's OK. Please push it on this PR. I'll invite other people to comment on the preview site.

Comment (rgrupesh) [reply src/pages/index.js:100]: Done.

**Review by kevin-wangzefeng — COMMENTED** [review]
Maybe layouts like https://cuelang.org/ is better. We basically need the project values ("why karmada") shown inside one page.

**Comment by RainbowMango** [comment]
@rgrupesh Can we organize the content as per @kevin-wangzefeng 's comments?

**Comment by RainbowMango** [comment]
@Arhell You can start to work on it now if you are still interested.

**Pull request merged**